# --- SDE-COPYRIGHT-NOTE-BEGIN ---
# This copyright note is auto-generated by ./scripts/Create-CopyPatch.
#
# Filename: package/.../libnfnetlink/0001-include-sys-types.h.patch
# Copyright (C) 2013 The OpenSDE Project
#
# More information can be found in the files COPYING and README.
#
# This patch file is dual-licensed. It is available under the license the
# patched project is licensed under, as long as it is an OpenSource license
# as defined at http://www.opensource.org/ (e.g. BSD, X11) or under the terms
# of the GNU General Public License as published by the Free Software
# Foundation; either version 2 of the License, or (at your option) any later
# version.
# --- SDE-COPYRIGHT-NOTE-END ---

include <sys/types.h> for u_int*_t types 

This issue was detected when using musl as default libc.

--- a/src/libnfnetlink.c	2013-01-01 17:16:25.926047486 +0100
+++ b/src/libnfnetlink.c	2013-01-01 17:16:46.990533842 +0100
@@ -45,6 +45,7 @@
 #include <netinet/in.h>
 #include <assert.h>
 #include <linux/types.h>
+#include <sys/types.h>
 #include <sys/socket.h>
 #include <sys/uio.h>
 
